tests/ui/test_ui_tools.py

```diff
@@ -1901,14 +1900,12 @@ def test_soup2markup(self, content, expected_markup, mocker):
                     "display_recipient": [
                         {
                             "is_mirror_dummy": False,
-                            "short_name": "aaron",
                             "email": "AARON@zulip.com",
                             "id": 1,
                             "full_name": "aaron",
                         },
                         {
                             "is_mirror_dummy": False,
-                            "short_name": "iago",
```

Comment:
This was removed at the same time as other short names, but since we have no typeddict for it then nothing is being removed from the api_types. We could definitely improve the typing of `display_recipient` in this way, maybe first to define what we have, though not necessarily.